.github/workflows/rules.yml

```diff
@@ -57,7 +65,10 @@ jobs:
     - name: Test with pytest core
       run: |
         pytest
-    - name: Test example 
+        if [ "${{ inputs.disable-jit }}" = true ]; then
+          NUMBA_DISABLE_JIT=1 pytest
+        fi
```

Comment:
I believe this one can be avoided altogether:
https://github.com/NNPDF/eko/blob/2a1a26d05b6a36454b8d1e1489e44b983a81a96d/pyproject.toml#L142

In this way, we do specialize the workflow as little as possible, and we stress the usage of local configuration files, that are read anyhow by the utilities, even in the CI.

Reply:
Well, we don't want to loose a pytest run with JIT enable, right?